src/Symfony/Component/Security/Http/Tests/RememberMe/PersistentRememberMeHandlerTest.php

```diff
@@ -118,7 +118,7 @@ public function testConsumeRememberMeCookieExpired()
         $this->tokenProvider->expects($this->any())
             ->method('loadTokenBySeries')
             ->with('series1')
-            ->willReturn(new PersistentToken(InMemoryUser::class, 'wouter', 'series1', 'tokenvalue', new \DateTime('-'.(31536000 - 1).' seconds')));
+            ->willReturn(new PersistentToken(InMemoryUser::class, 'wouter', 'series1', 'tokenvalue', new \DateTime('-'.(31536000 + 1).' seconds')));
```

Comment:
ping @nicolas-grekas as you made the switch to seconds in 15592293e7127e8c000f58d5f0a064f3ad863c6f